ext/standard/syslog.c

```diff
@@ -96,23 +96,9 @@ PHP_MINIT_FUNCTION(syslog)
 }
 /* }}} */
 
-PHP_RINIT_FUNCTION(syslog)
```

Comment:
Removed the rinit function because we already do `BG(syslog_device)=NULL` in the minit function, plus this is what causes the leak.